gazebo/10/ubuntu/bionic/libgazebo10/Dockerfile

```diff
@@ -1,7 +1,11 @@
 # This is an auto generated Dockerfile for gazebo:libgazebo10
 # generated from docker_images/create_gzclient_image.Dockerfile.em
 FROM gazebo:gzserver10-bionic
+# label gazebo packages
```

Comment:
Nit pick: could you tweak the templates so there is always a space before line comments.